src/dom/Value.ts

```diff
@@ -126,4 +135,141 @@ export interface Value {
  *
  * See: Value#get
  */
-export type PathElement = string | number;
\ No newline at end of file
+export type PathElement = string | number;
```

Comment:
The contents of the `DomValue.ts` file have been moved to `Value.ts`. The only changes are:
* The `DomValue` mixin was renamed `Value` so it would merge with the `Value` interface above.
* A `Value` namespace was set up at the bottom of the file to accommodate upcoming functionality.